### PR TITLE
feat: limit order book for all chains

### PR DESCRIPTION
**Pull request by dinhtienn**

### CODE REVIEW

File: src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx [hunk 1]
```diff
@@ -48,20 +58,33 @@ export default function OrderItem({
...
+      <ChainImage src={chain?.icon} alt="Network" />
       <Rate reverse={reverse}>{order.rate}</Rate>
-      <AmountInfo plus={reverse} amount={order.firstAmount} currency={currencyIn} upToSmall={upToSmall} />
-      <AmountInfo plus={!reverse} amount={order.secondAmount} currency={currencyOut} upToSmall={upToSmall} />
+      {(!upToSmall || !showAmountOut) && (
+        <AmountInfo plus={reverse} amount={order.firstAmount} currency={currencyIn} upToSmall={upToSmall} />
+      )}
+      {(!upToSmall || showAmountOut) && (
+        <AmountInfo plus={!reverse} amount={order.secondAmount} currency={currencyOut} upToSmall={upToSmall} />
+      )}
```
Comment (viet-nv) [inline src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx:87]: is this equivalent with:
```
{!upToSmall && <AmountInfo plus={showAmountOut  ? !reverse : reverse} amount={order.firstAmount} currency={currencyIn} upToSmall={upToSmall} />}

```

File: src/components/swapv2/LimitOrder/OrderBook/TableHeader.tsx [hunk 2]
```diff
@@ -17,32 +19,95 @@ const Header = styled(ItemWrapper)`
...
+          {upToSmall ? <br /> : ' '}
+          <Trans>({currencyIn?.symbol})</Trans>
+        </Text>
+      )}
+      <div>
+        <Flex>
+          <Text>
+            <Trans>AMOUNT</Trans>
+            {upToSmall ? <br /> : ' '}
+            <Trans>({!upToSmall || showAmountOut ? currencyOut?.symbol : currencyIn?.symbol})</Trans>
```
Comment (viet-nv) [inline src/components/swapv2/LimitOrder/OrderBook/TableHeader.tsx:94]: We dont need Trans for a token symbol

File: src/components/swapv2/LimitOrder/OrderBook/TableHeader.tsx [hunk 3]
```diff
@@ -17,32 +19,95 @@ const Header = styled(ItemWrapper)`
...
-      <Text>
-        <Trans>AMOUNT</Trans>
-        {upToSmall ? <br /> : ' '}
-        <Trans>({currencyOut?.symbol})</Trans>
-      </Text>
+      {!upToSmall && (
+        <Text>
+          <Trans>AMOUNT</Trans>
+          {upToSmall ? <br /> : ' '}
+          <Trans>({currencyIn?.symbol})</Trans>
```
Comment (viet-nv) [inline src/components/swapv2/LimitOrder/OrderBook/TableHeader.tsx:86]: We dont need Trans for a token symbol

File: src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx (hunk 1 above)
Comment (dinhtienn) [reply src/components/swapv2/LimitOrder/OrderBook/OrderItem.tsx:87]: No, It's not. But to make it easier to understand I will change it into
```
{!upToSmall ? (
    <>
      <AmountInfo plus={reverse} amount={order.firstAmount} currency={currencyIn} upToSmall={upToSmall} />
      <AmountInfo plus={!reverse} amount={order.secondAmount} currency={currencyOut} upToSmall={upToSmall} />
    </>
  ) : (
    <AmountInfo
      plus={showAmountOut ? !reverse : reverse}
      amount={showAmountOut ? order.secondAmount : order.firstAmount}
      currency={showAmountOut ? currencyOut : currencyIn}
      upToSmall={upToSmall}
    />
  )}
```

**Review by viet-nv — DISMISSED** [review]

**Pull request merged**